lucien-ios/AddAndEditNewComicBook/ComicFormViewController.swift

```diff
@@ -72,29 +77,65 @@ final class ComicFormViewController: UIViewController, AlertDisplaying {
+        storyTitleTextField.rx.text.orEmpty <->  comicFormViewModel.storyTitle >>> disposeBag
+        issueTextField.rx.text.orEmpty <->  comicFormViewModel.issue >>> disposeBag
+        publisherTextField.rx.text.orEmpty <->  comicFormViewModel.publisher >>> disposeBag
+        releaseDateTextField.rx.text.orEmpty <-> comicFormViewModel.release >>> disposeBag
+        seriesTitleTextField.rx.controlEvent([.editingChanged]).asObservable().subscribeNext { [weak self] in
+            self?.seriesTitleWarningLabel.isHidden = true
+            self?.seriesTitleTextField.borderColor = LucienTheme.dark
+            self?.checkIfAllRequiredFieldsAreFilled()
+        } >>> disposeBag
+        storyTitleTextField.rx.controlEvent([.editingChanged]).asObservable().subscribeNext { [weak self] in
+            self?.storyTitleWarningLabel.isHidden = true
+            self?.storyTitleTextField.borderColor = LucienTheme.dark
+            self?.checkIfAllRequiredFieldsAreFilled()
+        } >>> disposeBag
+        releaseDateTextField.rx.controlEvent([.editingChanged]).asObservable().subscribeNext { [weak self] in
+            guard let text = self?.releaseDateTextField.text else { return }
+            if text.characters.count > 4 {
+                self?.releaseDateTextField.deleteBackward()
+            }
+        } >>> disposeBag
+    }
+
+    private func configureUIPickerViewObservables() {
+        comicFormViewModel.genreTitles.bind(to: genrePicker.rx.itemTitles) { _, title in return title } >>> disposeBag
+        comicFormViewModel.conditionTitles.bind(to: conditionPicker.rx.itemTitles) { _, title in return title } >>> disposeBag
+        genrePicker.rx.itemSelected.subscribeNext { [weak self] row, _ in
+            let selectedGenre = Comic.Genre(rawValue: row)
+            self?.comicFormViewModel.genre = selectedGenre
+            self?.selectAGenreButton.setTitle(selectedGenre?.title, for: .normal)
+        } >>> disposeBag
+        conditionPicker.rx.itemSelected.subscribeNext { [weak self] row, _ in
+            let selectedCondition = Comic.Condition(rawValue: row)
+            self?.comicFormViewModel.condition = selectedCondition
+            self?.selectAConditionButton.setTitle(selectedCondition?.title, for: .normal)
+        } >>> disposeBag
     }
 
     private func fillFieldsWithComicFormViewModelData() {
-        guard let comicFormViewModel = comicFormViewModel else { return }
         if comicFormViewModel.comicFormMode == .edit {
-            seriesTitleTextField.text = comicFormViewModel.seriesTitle ?? ""
-            volumeTextField.text = comicFormViewModel.volume ?? ""
-            storyTitleTextField.text = comicFormViewModel.storyTitle
-            issueTextField.text = comicFormViewModel.issue ?? ""
-            publisherTextField.text = comicFormViewModel.publisher ?? ""
-            releaseDateTextField.text = comicFormViewModel.release ?? ""
+            seriesTitleTextField.text = comicFormViewModel.seriesTitle.value
```

Comment:
You shouldn't need to do this anymore - the bidirectional binding should fill it for you